python/cli/prelude_cli/views/build.py

```diff
@@ -312,15 +312,17 @@ def delete_detection(controller, detection):
 
 @build.command('create-threat-hunt')
 @click.argument('name')
+@click.option('-c', '--control', help='', required=True, type=click.Choice([c.name for c in Control]), case_sensitive=False)
```

Comment:
I see we do this in a few places, but this will include INVALID, NONE, SPLUNK, VECTR, INTUNE, and SERVICENOW as valid choices. I think we should be explicit about the supported controls

Reply:
yup limited to defender and crowdstrike for this command